.buildkite/pipelines/quality-gates/pipeline.test-qa.yaml

```diff
@@ -0,0 +1,15 @@
+steps:
+  - label: ":pipeline::kibana::seedling: Trigger Kibana Tests for ${ENVIRONMENT}"
+    command: echo "replace me with Kibana specific tests"
+    agent:
+      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
+
+  - label: ":pipeline::fleet::seedling: Trigger Fleet Kibana Tests for ${ENVIRONMENT}"
+    command: echo "replace me with Fleet specific Kibana tests"
+    agent:
+      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
```

Comment:
cc @juliaElastic this and the pipeline below for staging is where we may hook in the smoke pipeline you're working on. Please leave any feedback here if you see any problems.